core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -123,6 +123,13 @@ package object config {
         "Ensure that memory overhead is a double greater than 0")
       .createWithDefault(0.1)
 
+  private[spark] val DRIVER_LOG_LOCAL_DIR =
+    ConfigBuilder("spark.driver.log.localDir")
+      .doc("Specifies a local directory to write driver logs and enable Driver Log UI Tab.")
```

Comment:
nit enables

Reply:
The `enable` is connected to `to write ... and enable`.